node/full_node_test.go

```diff
@@ -9,52 +9,37 @@ import (
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/mock"
-	"github.com/stretchr/testify/require"
 
 	abci "github.com/cometbft/cometbft/abci/types"
-	"github.com/cometbft/cometbft/libs/log"
-	proxy "github.com/cometbft/cometbft/proxy"
-	"github.com/cometbft/cometbft/types"
 	"github.com/libp2p/go-libp2p/core/crypto"
 	"github.com/libp2p/go-libp2p/core/peer"
 
 	testutils "github.com/celestiaorg/utils/test"
 
-	"github.com/rollkit/rollkit/config"
 	"github.com/rollkit/rollkit/mempool"
 	"github.com/rollkit/rollkit/test/mocks"
 )
 
 // TestStartup checks if the node starts and stops without any errors
 func TestStartup(t *testing.T) {
-	assert := assert.New(t)
-	require := require.New(t)
-
 	ctx := context.Background()
-	node := initializeAndStartNode(ctx, require, assert)
-	defer cleanUpNode(node, assert)
+	node := initializeAndStartFullNode(ctx, t)
+	defer cleanUpNode(node, t)
 }
 
 func TestMempoolDirectly(t *testing.T) {
-	assert := assert.New(t)
-	require := require.New(t)
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
 
-	ctx := context.Background()
-	node := initializeAndStartNode(ctx, require, assert)
-	defer cleanUpNode(node, assert)
+	node := initializeAndStartFullNode(ctx, t)
+	assert := assert.New(t)
+	defer cleanUpNode(node, t)
 
```

Comment:
```suggestion
        defer cleanUpNode(node, t)
	assert := assert.New(t)
```
nit: interchange these lines so that `defer cleanUpNode(node, t)` happens right after node is initialized